LSP-Deno.sublime-settings

```diff
@@ -1,69 +1,47 @@
 {
-    "initializationOptions": {
-        "enable": true,
-        "lint": true,
-        "unstable": false
-    },
+    "enabled": false,
     "selector": "source.ts.react | source.tsx | source.ts | source.js.react | source.jsx | source.js",
-    "command": [
-        "deno",
-        "lsp"
-    ],
+    "command": ["deno", "lsp"],
     "settings": {
-        // Enables or disables the display of code lens information for implementations of items in
-        // the code.
+        // Enables or disables the display of code lens information for implementations of items in the code.
         "deno.codeLens.implementations": false,
-        // Enables or disables the display of code lens information for references of items in the
-        // code.
+        // Enables or disables the display of code lens information for references of items in the code.
         "deno.codeLens.references": false,
         // Enables or disables the display of code lens information for all functions in the code.
         "deno.codeLens.referencesAllFunctions": false,
-        // Enables or disables the display of code lenses that allow running of individual tests in
-        // the code.
+        // Enables or disables the display of code lenses that allow running of individual tests in the code.
         "deno.codeLens.test": true,
-        // Additional arguments to use with the run test code lens.  Defaults to `[ "--allow-all"
-        // ]`.
+        // Additional arguments to use with the run test code lens.  Defaults to `[ "--allow-all" ]`.
```

Comment:
The wrapping was quite unfortunate in some cases. Adjusted manually to not leave almost empty comment lines.